entries/p/paramountplus.com.json

```diff
@@ -0,0 +1,64 @@
+{
+	"Paramount+": {
+		"domain": "paramountplus.com",
+		"contact": {
+			"facebook": "ParamountPlus",
+			"twitter": "paramountplus"
+		},
+		"keywords": [
+			"entertainment"
+		],
+		"regions": [
+			"al",
+			"ad",
+			"ar",
+			"au",
+			"at",
+			"bo",
+			"ba",
+			"br",
+			"bg",
+			"ca",
+			"cl",
+			"co",
+			"cr",
+			"hr",
+			"cz",
+			"dk",
+			"do",
+			"ec",
+			"sv",
+			"fi",
+			"fr",
+			"de",
+			"gt",
+			"hn",
+			"hk",
+			"hu",
+			"in",
+			"it",
+			"mx",
+			"me",
+			"nl",
+			"ni",
+			"no",
+			"pa",
+			"py",
+			"pe",
+			"pl",
+			"pt",
+			"ro",
+			"rs",
+			"sk",
+			"si",
+			"kr",
+			"es",
+			"se",
+			"ch",
+			"gb",
+			"us",
+			"uy",
+			"ve"
```

Comment:
I only see that they support the following countries:
```suggestion
      "ar",
      "au",
      "bo",
      "br",
      "cl",
      "co",
      "cr",
      "de",
      "do",
      "ec",
      "fi",
      "gt",
      "hn",
      "ni",
      "no",
      "pa",
      "pe",
      "py",
      "se",
      "sv",
      "uy",
      "ve"
```
source: https://www.paramountplus.com/global/
![screenshot](https://user-images.githubusercontent.com/3535780/161440682-7eb73b1f-5ed4-450b-9064-2a5d8cc10ac4.png)

Reply:
@fletcherdodd you marked this as resolved without updating any code :thinking:

Reply:
@fletcherdodd you marked this as resolved without updating any code :thinking: